requirements/ingest/embed-voyageai.txt

```diff
@@ -0,0 +1,140 @@
+#
+# This file is autogenerated by pip-compile with Python 3.11
```

Comment:
Our dependencies are compiled using Python 3.9 to make sure we're compatible with 3.9 - 3.12. Could you rerun `make pip-compile` from the root directory of the project using Python 3.9?